### DISCUSSION TITLE
How long will Next.js 12 receive updates?

### DISCUSSION THREAD

**Discussion by CalebJamesStevens** [vercel/next.js#56813] [Help]
### Summary

My team is trying to figure out how long next.js 12 will receive security updates so we can prioritize upgrading.

**Comment by kenreis** [comment]
My team needs to know this as well

**Comment by ctopheryoung** [comment]
Any updates on this? Surprisingly hard to find any info on EOL plans around v12

**Comment by YasielCabrera** [comment]
I asked on discord and this is the answer:

> Major versions aren't receiving any updates after a newer one has been released as far as I know 
> So v12 is technically a year EOL

According to that info, now that Next 14 has been released, the version 13 reached EOL too. 

EDIT:
Also found this resource: https://endoflife.date/nextjs

**Reply by hysan** [reply]
@leerob Would is be possible for Vercel to add this EOL policy to the official documentation? Either this quote or whatever is the actual policy?

**Reply by leerob** [reply]
We're working on publishing an LTS policy. More to come soon.
